gridLang/GridLangConstants.ts

```diff
@@ -106,6 +106,23 @@ export const IntegerCellDef: CellDef = {
     parse: (value: any) => parseInt(value),
 }
 
+export const IntegersCellDef: CellDef = {
+    keyword: "",
+    cssClass: "IntegerCellDef",
+    description: "",
+    regex: /^-?[0-9 ]+$/,
```

Comment:
This regex allows for a single negative number, but not multiple negative numbers.
One way to do that properly would be (this also makes the regex more strict, in that there is only exactly one space between numbers, and there are no leading and trailing spaces):
```suggestion
    regex: /^(-?\d+( |$))+$/,
```

See https://regexr.com/7g0rc

Reply:
Should we maybe disallow negative numbers altogether since indicator ids can't be negative?

Reply:
Yes that also makes sense!